core-blocks/image/index.js

```diff
@@ -165,6 +170,10 @@ export const settings = {
 	save( { attributes } ) {
 		const { url, alt, caption, align, href, width, height, id } = attributes;
 
+		const classes = classnames( align ? `align${ align }` : null, {
+			'is-resized': !! width || !! height,
```

Comment:
Just curious, could we remove the need for this class with the following?

```css
img[height],
img[width] {
  ...
}
```

Reply:
And if caption needs targeting.

```css
img[height] + figcaption,
img[width] + figcaption {
  ...
}
```

Reply:
This change needs a `deprecated` version though :)

Reply:
Added a deprecated version in https://github.com/WordPress/gutenberg/pull/6496/commits/68b40e506682d788b9dbf8008fbaeeb66b19687f. Did I do that right?

To clarify what happens — if the figure is as wide as the main content, it needs `fit-content`. If the figure is smaller than the main content, it needs `min-content` so that a long caption doesn't make the figure wider than its smallest element (which will always be the image).

Should the class be called "is-resized" for that, or can we think of something better?